CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 0.10.14 - 2025-07-02
```

Comment:
I think that we should release this as 0.11.0 considering it's introducing an entirely new feature.